src/main/resources/application.properties

```diff
@@ -26,7 +26,7 @@ gitlab.deploy.key=${DEPLOY_KEY:0}
 
 # config 
 config.repository.id=${CONFIG_REPOSITORY_ID:1}
-config.file=${CONFIG_FILE:schema/config.yml}
+config.file=${CONFIG_FILE:/schema/config.yml}
 webhook.file=${WEBHOOK_FILE:/schema/webhooks.yaml}
```

Comment:
can we update the webook and config default values. I think they are at 'runtime' instead of 'schema' now.

Reply:
Updated to `/runtime` instead of `/schema`